spec/openid_connect/response_object/id_token_spec.rb

```diff
@@ -224,6 +224,67 @@
     end
   end
 
+  describe '.decode_with_keys' do
```

Comment:
These tests will fail until json-jwt gem version is updated as IdToken uses JSON::JWT.decode_with_keys